app/models/user.rb

```diff
@@ -118,6 +118,9 @@ def has_favorite2?(item)
     !! @favorites.member?([item.id, item.class.to_s])
   end
 
+  scope :pro_expires_this_month, -> (range) { where(pro_expires_at: Date.today.all_month) }
```

Comment:
Oh, hah, I accidentally left that in when I changed it from `pro_expires_in` to this more specific, more-lazy solution.  I'll go fix that